src/python/turicreate/_deps/minimal_import.py

```diff
@@ -0,0 +1,15 @@
+# Copyright © 2020 Apple Inc. All rights reserved.
```

Comment:
Why are we adding entirely new files at this point in the review process? What is the point of this file?

Reply:
I need the function defined and the flag `USE_MINIMAL`.

Reply:
Please explain a bit more. That doesn't really answer my question.

Given that this pull request is almost ready to go merge, perhaps it would be better to add this file to your other pull request.

Reply:
please check #3175.